charts/prometheus-mysql-exporter/templates/servicemonitor.yaml

```diff
@@ -30,6 +30,38 @@ spec:
 {{ toYaml . | trim | indent 4 -}}
   {{- end }}
   endpoints:
+  {{- if .Values.serviceMonitor.multipleTarget }}
+  {{- range  .Values.serviceMonitor.targets }}
+    - path: /probe
+      port: {{ $.Values.service.name }}
+  {{- if $.Values.serviceMonitor.interval }}
+      interval: {{ $.Values.serviceMonitor.interval }}
+  {{- end }}
+  {{- if $.Values.serviceMonitor.scrapeTimeout }}
+      scrapeTimeout: {{ $.Values.serviceMonitor.scrapeTimeout }}
+  {{- end }}
+      metricRelabelings:
+        - action: replace
+          replacement: {{ .endpoint }}
+          sourceLabels: [instance]
+          targetLabel: instance
+        - action: replace
+          replacement: {{ .name }}
+          sourceLabels: [target]
+          targetLabel: target
+      {{- if $.Values.serviceMonitor.metricRelabelings -}}
+        {{ toYaml $.Values.serviceMonitor.metricRelabelings | nindent 8 }}
+      {{- end }}
+  {{- if $.Values.serviceMonitor.relabelings }}
+      relabelings: {{ toYaml $.Values.serviceMonitor.relabelings | nindent 8 }}
+  {{- end }}
+      params:
+        target:
+          - {{ .endpoint }}:{{ .port | default 3306 }}
+        auth_module:
+          - client.{{ .name }}
```

Comment:
Can we possibly add the feasibility of using shared credentials?
```        
        {{- if .Values.serviceMonitor.sharedSecret }}
        auth_module:
          - client.servers
        {{- else }}
        auth_module:
          - client.{{ .name }}
        {{- end }}
```

Reply:
Implemented using `.Values.serviceMonitor.multiTarget.sharedSecret.enabled` and `.Values.serviceMonitor.multiTarget.sharedSecret.name`. Is this what you were expecting ?

Reply:
Yes. Thank you!